docs/virtual_machines/interfaces_and_networks.md

```diff
@@ -557,6 +557,23 @@ spec:
 > **Note:** The IPv6 address for the VM and default gateway **must** be the ones
 > shown above.
 
+### masquerade - IPv6 single-stack support
+
+`masquerade` mode can be used in IPv6 single stack clusters to provide a VM
+with an IPv6 only connectivity.
+
+As with the IPv4 `masquerade` mode, the VM can be contacted using the pod's IP
+address - which will be in this case the IPv6 one.
+Outgoing traffic is also "NAT'ed" to the pod's respective IPv6 address.
+
+As with the dual-stack cluster, the configuration of the IPv6 address and the default route is
+not automatic; it should be configured via cloud init, as shown in the [dual-stack section](#masquerade).
```

Comment:
I do not know if it is possible, but the section should be a different one: `masquerade - IPv4 and IPv6 dual-stack support`.

Maybe `masquerade-IPv4-and-IPv6-dual-stack-support`, it needs testing.

Reply:
done